Controllers/BlogController.cs

```diff
@@ -35,4 +35,40 @@ public async Task<ActionResult<Blog>> GetBlogByID(int id)
         }
         return Ok(_mapper.Map<Blog>(blog));
     } 
+
+    [HttpPost]
+    public async Task<ActionResult<Blog>> CreateBlogPost(BlogCreation post)
+    {
+        var fblog = _mapper.Map<Entities.Blog>(post);
+
+        await _blogService.CreateBlogPost(fblog);
```

Comment:
It would be better for you to return the blog post that was created. It would be unexpected for any changes to the blog post to come back in the fblog reference.